client/src/components/Auth/Registration.tsx

```diff
@@ -1,12 +1,32 @@
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
 import { useForm } from 'react-hook-form';
 import { useRegisterUserMutation, TRegisterUser } from '~/data-provider';
-import { SHOW_GOOGLE_LOGIN_OPTION, DOMAIN_SERVER } from '~/utils/envConstants';
+import { SHOW_GOOGLE_LOGIN_OPTION, ALLOW_OPENID, OPENID_LABEL, DOMAIN_SERVER } from '~/utils/envConstants';
 
 function Registration() {
   const navigate = useNavigate();
 
+  const [openidLogo, setOpenidLogo] = useState('');
+  const openidLabel = typeof OPENID_LABEL !== 'undefined' ? OPENID_LABEL : 'OpenID';
+
+  useEffect(() => {
+    fetch('/images/openid/logo.svg')
+      .then((response) => {
+        if (response.ok) {
+          return response.url;
+        } else {
+          throw new Error('Logo not found');
+        }
+      })
+      .then((logoUrl) => {
+        setOpenidLogo(logoUrl);
+      })
+      .catch((error) => {
+        console.log('Error:', error);
+      });
+  }, []);
```

Comment:
can we just include the svg in the project manually? We shouldn't really fetch an image like this

Reply:
Could it be a url reference? It could use a default logo if a URL is not given. Again this is connected the label response. By putting our logo on the button it directs our users to naturally use it.  We are working with students and some are less savvy on their computer skills. So anything we can do to make it easier for staff, faculty, and student means less calls to our service desk.

Reply:
btw, I'm finished with the work on that now, so just waiting for it to be reviewed.